main.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+long long factorial (int n)
+{
+	if(n==0 || n==1)
+	{
```

Comment:
We usually use two or 4 spaces to indent code

Reply:
Thank you for the review, code is changed accordingly.